modules/solvers/voltage-divider.stanza

```diff
@@ -25,23 +25,27 @@ public defstruct VoltageDividerSolution :
   vo: Toleranced
 
 public defn voltage-divider-solver (v-in: Toleranced,
-                             v-out: Toleranced,
-                             current: Double) -> VoltageDividerSolution|False:
+                                    v-out: Toleranced,
+                                    current: Double) -> VoltageDividerSolution|False:
```

Comment:
Should we also accept current as a Toleranced as well? `current: Toleranced|Double,`

There are situations where I want to spec a min and a max current through my voltage divider.

Reply:
Sure I'll support a Toleranced current. Perhaps `current` should always be Toleranced?

Reply:
Sure -- that's a bigger change but that is a better long term formulation.

Reply:
Is the current value a rating?